src/Components/CommoditiesContainer/CommoditiesContainer.test.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable react/prop-types */
+import { render, screen } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import configureStore from 'redux-mock-store';
+import CommoditiesContainer from './CommoditiesContainer';
+
+// Mock the redux store
+const mockStore = configureStore([]);
+const initialState = {
+  commodityStore: {
+    commodities: [
+      {
+        symbol: 'ABC',
+        name: 'Gold',
+        currency: 'USD',
+        stockExchange: 'NYSE',
```

Comment:
Good job working on the project requirements but i noticed you disabled eslint in this file it would be nice if you fixed this by enabling eslint because it is against microverse policy to disable linters because linters help us write clean code.